mods/key_value.js

```diff
@@ -77,7 +77,9 @@ KVBucket.prototype.createBucket = function(restbase, req) {
     if (!Array.isArray(opts.compression)) {
         opts.compression = [opts.compression];
     }
-    opts.updates = opts.updates || undefined;
+    opts.updates = {
```

Comment:
Maybe still allow users to specify their own updates instead of forcing timeseries on them?

```js
opts.updates = opts.updates || { pattern: 'timeseries' }
```